### PR TITLE
Allow `ember-template-lint` v5

### LINKED ISSUE #236: Support ember-template-lint v5
First alpha of ember-template-lint v5 is out: https://github.com/ember-template-lint/ember-template-lint/releases/tag/v5.0.0-alpha.0

The work is being tracked in: https://github.com/ember-template-lint/ember-template-lint/issues/2319

### PR DESCRIPTION
**Pull request by bertdeblock**
Closes #236.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -53,6 +53,25 @@ jobs:
...
 
+  ember-template-lint:
+    name: ember-template-lint@${{ matrix.etlVersion }}
+    needs: [test]
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        etlVersion: [^4.0.0, 5.0.0-alpha.0]
```
Comment (bertdeblock) [inline .github/workflows/ci.yml:64]: Will update to `etlVersion: [^4.0.0, ^5.0.0]`, once there is a stable release.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -53,6 +53,25 @@ jobs:
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: volta-cli/action@v1
```
Comment (bmish) [inline .github/workflows/ci.yml:68]:
```suggestion
      - uses: volta-cli/action@v4
```

Comment (bmish) [inline .github/workflows/ci.yml:68]:
```suggestion
      - uses: actions/checkout@v3
      - uses: volta-cli/action@v4
```

Comment (bertdeblock) [reply .github/workflows/ci.yml:68]: This just uses the same versions as the other jobs, I could make a separate PR to bump everything and then rebase this one?

Comment (bertdeblock) [reply .github/workflows/ci.yml:68]: Made #238.

Comment (bmish) [reply .github/workflows/ci.yml:68]: Yep, separate PR is fine.

**Review by bmish — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bmish) [reply .github/workflows/ci.yml:64]: https://github.com/ember-template-lint/ember-template-lint-plugin-prettier/pull/245

**Pull request merged**